src/odb/src/db/dbAccessPoint.cpp

```diff
@@ -166,6 +168,8 @@ dbIStream& operator>>(dbIStream& stream, _dbAccessPoint& obj)
   stream >> obj.bpin_;
   stream >> obj.accesses_;
   stream >> obj.iterms_;
+  stream >> obj.vias_;
+  stream >> obj.path_segs_;
```

Comment:
@Mahfouz-z move inside the user code section

Reply:
Code produced via generator now.